src/ContainerDetails.jsx

```diff
@@ -38,6 +38,14 @@ const ContainerDetails = ({ container }) => (
         <dd>{render_container_state(container)}</dd>
         <dt>{_("Ports")}</dt>
         <dd>{render_container_published_ports(container.Ports)}</dd>
+        <dt>{_("IP Address")}</dt>
+        <dd>{containerDetail ? containerDetail.NetworkSettings.IPAddress : ""}</dd>
+        <dt>{_("IP Prefix Length")}</dt>
+        <dd>{containerDetail ? containerDetail.NetworkSettings.IPPrefixLen : ""}</dd>
+        <dt>{_("Gateway")}</dt>
+        <dd>{containerDetail ? containerDetail.NetworkSettings.Gateway : ""}</dd>
+        <dt>{_("Mac Address")}</dt>
```

Comment:
Let's please spell it "MAC", it's not your Scottish uncle :wink:

Reply:
Aha yeah,  but should I do these changes? Quite confused (@marusak has done some commits

Reply:
Please go ahead, but squash/amend these changes into the last commit please

Reply:
Wow that makes me more confused. Not very skilled at git, what should I do? For now I just pull the `origin master` and it tell me that I'm merging. Then? reset? delete commit? And what does it mean to `squash/amend` changes? 😂 Totally no idea man.

Reply:
@EricJeffrey : When in doubt, just add fixup commits on top, and Matej or me are happy to clean them up.

"squash/amend" essentially means to rewrite the history and change existing commits, using `git commit --amend` or `git rebase -i`. We don't want "do this in a broken way", "fix it", "fix it  harder" separate commits on master, that makes archaeology, bisecting, and backporting changes too hard.

Reply:
Are you using commandline to work with git or some graphical tool?
If the former then:
1. `git reset --hard HEAD~5` (in case you already pulled and now you have mess there)
2. `git pull` (now you should have exactly the same as is here, the same 2 commits with the same sha) (assuming you have set up remote tracking, if not, then `git pull origin network_info`)
3. Do the changes
4. `git add -u`
5. `git commit --amend` (now one commit should be the same and the other should have different sha)(it will open text editor, just save it without any changes)
6. `git push -f`

Reply:
Thanks, Guess it work now?

Reply:
👍 You guys are amazing

Reply:
Oh hell.. I will fix it right now 😂

Reply:
Good work with `git` but not the change Martin had in mind :D At least you can try this git dance once more ;)

Reply:
It should be `MAC Address`

Reply:
Now what? This is interesting. Thanks really

Reply:
Heh, thanks @EricJeffrey ! Third time's the charm!